lib/openstack_query/api/query_api.py

```diff
@@ -150,20 +150,41 @@ def run(
         self._query_run = True
         return self
 
-    def to_list(
-        self, as_objects=False, flatten=False, groups: Optional[List[str]] = None
-    ) -> Union[Dict[str, List], List[Dict[str, str]]]:
+    def to_objects(
+        self, groups: Optional[List[str]] = None
+    ) -> Union[Dict[str, List], List]:
         """
-        Public method to return results as a list (ungrouped) or dict (if grouped/flattened)
-        :param as_objects: if true return result as openstack objects
-        :param flatten: boolean which will flatten results if true
-        :param groups: a list group to limit output by
+        Public method to return results as openstack objects
```

Comment:
`this is either returned as a list if no groups are specified, or as a dict if they grouping was requested`